src/swap/components/Swap.tsx

```diff
@@ -50,10 +53,14 @@ export function Swap({
         )}
         data-testid="ockSwap_Container"
       >
-        <div className="mb-4">
-          <h3 className={text.title3} data-testid="ockSwap_Title">
+        <div className="mb-4 flex items-center justify-between">
```

Comment:
@cpcramer have you tested how this looks with and without `swapSettings`, so that the CSS is not breaking.

Reply:
Yes, it works as expected!